src/loginServices.js

```diff
@@ -110,15 +115,50 @@ export function getBindings(okapiUrl, store, tenant) {
     });
 }
 
+function getErrorMessage(error = {}) {
+  const { code = 'default.error', parameters } = error;
+  const values = parameters ?
+    parameters.reduce(
+      (res, key) => {
+        res[key.key] = key.value;
+        return res;
```

Comment:
Please add an empty line before`return` statement or even refactor it to the code below:
```
parameters.reduce((res, { key, value }) => { ...res, [key]: value }, {})
```

Try to avoid such name constructions like `key.key` whenever possible.